web_src/css/repo.css

```diff
@@ -1913,12 +1913,14 @@
 }
 
 
-.repository.packages .empty {
+.repository.packages .empty,
+.repository.actions .empty {
   padding-top: 70px;
   padding-bottom: 100px;
 }
 
-.repository.packages .empty .svg {
+.repository.packages .empty .svg,
+.repository.actions .empty .svg {
   height: 48px;
```

Comment:
`{{svg "octicon-no-entry" 32}}` in code but `height: 48px;` here?

I think the `empty` CSS class doesn't make sense.

Reply:
Yes we should render these SVGs correctly in HTML, and remove this resizing here. This approach goes for all images and SVGs, their size should be correct in HTML, making CSS resize unneeded.

Reply:
This is removed https://github.com/go-gitea/gitea/pull/25118/commits/738df8f18f8515a2c3c82483300e4570d9307e57